build.gradle.kts

```diff
@@ -74,6 +74,7 @@ dependencies {
     implementation("io.ktor:ktor-client-content-negotiation:$ktorVersion")
     implementation("io.ktor:ktor-client-websockets:$ktorVersion")
 
+    implementation("org.ehcache:ehcache:3.9.11")
```

Comment:
I see there is v 3.10.8. Would that be better?

Reply:
SUre - we can move to the more recent.  I think I sorted by dates and 3.9.11 was the latest pushed to maven Central (feb 15, 2023 vs Nov of 2022)

Reply:
updated